recipes/qt/6.x.x/conanfile.py

```diff
@@ -433,7 +433,7 @@ def requirements(self):
             self.requires("md4c/0.4.8")
 
     def build_requirements(self):
-        self.tool_requires("cmake/3.25.0")
+        self.tool_requires("cmake/3.25.2")
```

Comment:
```suggestion
        if not self._cmake_new_enough("3.16"):
            self.tool_requires("cmake/3.25.2")
```

Reply:
I like this idea, but I saw @prince-chrismc ask to avoid adding this check "too much" before v2: https://github.com/conan-io/conan-center-index/pull/15735#discussion_r1101825541

Reply:
:)